clients/admin-ui/src/features/datastore-connections/add-connection/forms/ConnectorParametersForm.tsx

```diff
@@ -368,7 +368,8 @@ const ConnectorParametersForm: React.FC<ConnectorParametersFormProps> = ({
               getFormField(key, item)
             )}
             <ButtonGroup size="sm" spacing="8px" variant="outline">
-              <Button
+              {/* Temporarily hiding this button */}
```

Comment:
@LKCSmith Should this be a conditional expression such as `!connection?.key` added to render the Test connection button if the user is updating an existing connection?

Reply:
User cannot test a connection when editing now as a result if this is commented out.

Reply:
@mfbrown - this is what the create a connection screen looks like without the button. There's not too much context in the issue, so I am assuming we want to hide that button for now under all circumstances, temporarily?
<img width="684" alt="Screen Shot 2023-01-10 at 1 56 54 PM" src="https://user-images.githubusercontent.com/99051851/211649909-8e83b608-8b2b-48c7-9f61-b8ae816bc275.png">

Reply:
I would prefer to implement what @chriscalhoun1974 is describing where the test connection button is hidden (or preferably disabled!!) if the connector hasn't been created yet, but I am conscious that's a little extra scope than what we discussed here. Do you think it can be done in this sprint? @LKCSmith 

cc: @Kelsey-Ethyca

Reply:
@mfbrown - Sure, I'll take a look at it!

Reply:
@mfbrown  @chriscalhoun1974 - correct me if I'm wrong, but it looks like the behavior you're both requesting already exists?

https://user-images.githubusercontent.com/99051851/212020410-2586c442-71bc-4a79-8e69-2ef249f9ba98.mov


Let me know if i'm missing something or if I can close this issue.

Reply:
@LKCSmith @mfbrown I believe there is some confusion on what the best overall user experience is when a user is creating a connection. @LKCSmith Is correct in that the Test Connection is disabled until a user has created a connection record by clicking the Save button. However via Slack, I believe there was a suggestion to hide the Test Connection button instead of disabling it.